lenskit-core/specs/data-manifest.md

```diff
@@ -0,0 +1,111 @@
+# Data Manifest Schema
+
+This document describes the format of data manifests.
+
+Data manifests are JSON-style data.  Currently they are generally described in YAML files, but they may be described in other formats such as JSON.
```

Comment:
YAML is a superset of JSON, and we don't use any of its semantic extensions, only its syntactic ones. Is there a better way to discuss this?